lib/roby/app/scripts/shell.rb

```diff
@@ -103,7 +104,7 @@ def gets
     return []
 end
 
-class ShellEvalContext < BasicObject
+class ShellEvalContext
     include ::Kernel
```

Comment:
Is there a reason to keep including `Kernel` now that `ShellEvalContext` is not just a basic object anymore?

Reply:
Nope. Good catch.